lang/cs/Org.Apache.REEF.Driver/Task/IRunningTask.cs

```diff
@@ -31,13 +31,7 @@ public interface IRunningTask : IIdentifiable, IDisposable
         /// <summary>
         /// the context the task is running on.
         /// </summary>
-        IActiveContext ActiveContext { get; set; }
-
-        /// <summary>
-        /// Sends the message to the running task.
-        /// </summary>
-        /// <param name="message"></param>
-        void OnNext(byte[] message);
```

Comment:
It never was implemented and has the exact same meaning as `Send`, which was implemented.